md-dao/src/main/java/com/wipro/ats/bdre/md/dao/GetDotStringDAO.java

```diff
@@ -41,7 +41,10 @@
     private static final Logger LOGGER = Logger.getLogger(GetDotStringDAO.class);
     @Autowired
     SessionFactory sessionFactory;
-
+    private static final  String PROCESSID="processId";
+    private static final String LINEAGEQUARYID="lineageQuery.queryId";
+    private static final String DOTSTRING="dotString";
+    private static final String NODEID="nodeId";
     public List<String> getDotString(Integer processId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "getDotString" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)